openlibrary/templates/type/work/editions_datatable.html

```diff
@@ -14,18 +14,20 @@
         $ edition_list_start = time()
         $ editions = editions or work.get_sorted_editions()
         $ index_padding = len(str(len(editions)))
+        $ render_first_flag = 1
         $for book in editions:
             $ book_keys.append(book['key'].replace('/books/', ''))
             $ edition_sort_key = str(loop.index0+1).zfill(index_padding)
-            $if book.key == edition.key:
-              <tr class="highlight">
-                $# This enables us to always render a select edition first in the table (default)
-                $:render_template("books/edition-sort", book, edition_sort_key, render_first=True)
-              </tr>
-            $else:
-              <tr>
-                $:render_template("books/edition-sort", book, edition_sort_key)
-              </tr>
+            $if book.key != edition.key:
+              $if render_first_flag == 1:
+                <tr class="highlight">
+                  $:render_template("books/edition-sort", book, edition_sort_key, render_first=True)
+                </tr>
+                $ render_first_flag = 0
+              $else:
+                <tr>
+                  $:render_template("books/edition-sort", book, edition_sort_key)
+                </tr>
```

Comment:
```suggestion
            $if book.key != edition.key:
              <tr>
                $:render_template("books/edition-sort", book, edition_sort_key)
              </tr>
```
No need to render the row corresponding to the page that is being viewed.